R/truncate_triangles.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' This function ingests a reporting triangle/matrix and the number of
 #'   truncated reporting triangles we want to create, `n`, and iteratively
-#'   truncated the reporting triangle, working from bottom to top for `n`
+#'   truncates the reporting triangle, working from the latest reference time
```

Comment:
These updates are all fine but each of these 3 functions describes the same action using different language. These kinds of things add user friction and so it is best to standardise where possible. Related to #144.

Reply:
Related to #144, I went through these functions and clarified when something is a reporting triangle vs when the function expects a truncated reporting triangle, so that we can use @inheritParams as intended.